DoublePendulum/DoublePendulum.js

```diff
@@ -0,0 +1,53 @@
+import { WIDTH, HEIGHT } from "../sketchlib/dimensions.js";
+
+export const sketch = (p) => {
+  p.setup = () => {
+    p.createCanvas(
+      WIDTH,
+      HEIGHT,
+      undefined,
+      document.getElementById("sketch-canvas")
+    );
+  };
+
+  p.draw = () => {
```

Comment:
How do I want to improve upon the visualizations from the old sketch?

- particle trails?
- Make a bunch of double pendulums with slightly different positions?
- Allow interactivity? Click to grab one of the pendula to set a new start position?

Reply:
Added these to the notes section of the lab